README.md

```diff
@@ -1,4 +1,7 @@
 # HW06
+
+I'm doing my own thing for this homework. My goal with this Rscript is to design an antisense oligonucleotide with a catalytic core that will bind a target sequence.
+______________________________________________________________________________________________
```

Comment:
Similarly, I decided to do my own thing for this homework. I was having trouble with importing/formatting the HW6 dataset.